documentation/tutorials/get-started-with-phoenix.md

```diff
@@ -0,0 +1,467 @@
+4. A terminal to run the examples
+
+## Steps
+
+For this tutorial, we'll use examples based around creating a help desk.
+
+We will make the following resources:
+
+- `PhoenixHelpdesk.Support.Ticket`
+- `PhoenixHelpdesk.Support.Representative`
+
+The actions we will be able to take on these resources include:
+
+- Opening a new Ticket
+- Closing a Ticket
+- Assigning a Ticket to a representative
+
+### Create a New Phoenix Project
+
+_This section is based the [Phoenix installation docs](https://hexdocs.pm/phoenix/installation.html). For more details go there._
+
+We first need to create a fresh phoenix project using the phoenix project generator.
+
+```bash
+mix archive.install hex phx_new
+```
+
+And then use the project generator.
+
+```bash
+mix phx.new phoenix_helpdesk
+```
+
+This will then generate a project boilerplate and ask if you want to fetch and install dependencies. **Answer 'Yes' to 'Fetch and install dependencies? [Yn]'**.
+
+Your terminal should now look very similar to this:
+
+```bash
+* creating phoenix_helpdesk/priv/static/images/phoenix.png
+* creating phoenix_helpdesk/priv/static/favicon.ico
+
+Fetch and install dependencies? [Yn] y
+* running mix deps.get
+* running mix deps.compile
+
+We are almost there! The following steps are missing:
+
+    $ cd phoenix_helpdesk
+
+Then configure your database in config/dev.exs and run:
+
+    $ mix ecto.create
+
+Start your Phoenix app with:
+
+    $ mix phx.server
+
+You can also run your app inside IEx (Interactive Elixir) as:
+
+    $ iex -S mix phx.server
+```
+
+Please `cd` into the project directory **BUT DO NOT RUN `mix ecto.create`.** We first need to integrate Ash and AshPostgres into our project.
+
+### Add Dependencies
+
+We now need to add Ash, AshPhoenix and AshPostgres to our Phoenix project.
+
+First lets add the dependencies to the `deps` function in our `mix.exs`:
+
+```elixir
+# mix.exs
+
+  def deps do
+    [
+      {:phoenix, "~> 1.6.15"},
+      # ...
+      {:ash, "~> 2.5.10"},
+      {:ash_postgres, "~> 1.3.6"},
+      {:ash_phoenix, "~> 1.1"},
+      # If using ElixirLS then including elixir_sense enabled ash auto-complete
+      {:elixir_sense, github: "elixir-lsp/elixir_sense", only: [:dev, :test]}
+    ]
+  end
+```
+
+Each Ash package also has its own formatter extension, include these in your `.formatter.exs` to ensure consistent formatting.
+
+```elixir
+# .formatter.exs
+[
+  import_deps: [:ecto, :phoenix, :ash, :ash_phoenix, :ash_postgres],
+  inputs: ["*.{ex,exs}", "priv/*/seeds.exs", "{config,lib,test}/**/*.{ex,exs}"],
+  subdirectories: ["priv/*/migrations"]
+]
+```
+
+Now in the terminal install these new dependencies:
+
+```bash
+mix deps.get
+```
+
+### If using Phoenix 1.7 or higher
+
+we need to change `helpers: false` to `helpers: true` in the router section:
+
+```elixir
+# lib/example_web.ex
+
+defmodule PhoenixHelpdeskWeb do
+# ...
+  def router do
+    quote do
+      use Phoenix.Router, helpers: true # <-- Change this line
```

Comment:
This is only required for AshAuthenticationPhoenix. Doesn't mean we shouldn't have it here, but maybe good to clarify that its in preparation for that?

Reply:
Oh nice, in that case I'll cut it! 👍